packages/calcite-components/src/components/button/button.scss

```diff
@@ -72,16 +152,222 @@
   }
 }
 
+.icon {
+  @apply relative
+    m-0
+    inline-flex
+    font-normal;
+  line-height: inherit;
+  color: var(--calcite-button-icon-color, currentColor);
+}
+
+// fab variants
+:host([round]) {
+  --calcite-internal-button-corner-radius: 50px;
```

Comment:
Sidebar: Do we need to move fab-specific styles to `fab.scss` (similar to the overrides applied by `inline-editable.scss` below)? If so, we can tackle that separately.

Reply:
I moved this over to FAB but kept the rest